core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -596,7 +597,16 @@ protected void onModified() throws IOException {
      * TCP agent port.
      * 0 for random, -1 to disable.
      */
-    private int slaveAgentPort = SystemProperties.getInteger(Jenkins.class.getName()+".slaveAgentPort",0);
+    private int slaveAgentPort = getSlaveAgentPortInitialValue(0);
```

Comment:
Hmm. We were using -1 in SetupWizard as a default value. Is it just another inconsistency?